### PR TITLE
[Website] Add release post for ADBC 15

### PR DESCRIPTION
**Pull request by lidavidm**

### CODE REVIEW

File: _posts/2024-11-13-adbc-15-release.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+     3	eitsupi
+     2	Matt Topol
+     2	Tornike Gurgenidze
+     2	qifanzhang-ms
+     1	Sudhir Reddy Emmadi
+```
+
+## Roadmap
+
+<!-- TODO: fill this portion in. -->
```
Comment (kou) [inline _posts/2024-11-13-adbc-15-release.md:80]: Do you want to update this section before we publish this? Or should we remove this section?

**Review by kou — APPROVED** [review]
+1

File: _posts/2024-11-13-adbc-15-release.md (hunk 1 above)
Comment (lidavidm) [reply _posts/2024-11-13-adbc-15-release.md:80]: ...hmm, I must not have committed my changes

Comment (lidavidm) [reply _posts/2024-11-13-adbc-15-release.md:80]: oh d'oh, I only did the release highlights

Comment (lidavidm) [reply _posts/2024-11-13-adbc-15-release.md]: updated

File: _posts/2024-11-13-adbc-15-release.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 15 release of
+the Apache Arrow ADBC libraries. This covers includes [**31
+resolved issues**][1] from [**13 distinct contributors**][2].
```
Comment (amoeba) [inline _posts/2024-11-13-adbc-15-release.md:29]: This language has been used in previous release blog posts but I just now noticed it's slightly awkward.

```suggestion
The Apache Arrow team is pleased to announce the version 15 release 
of the Apache Arrow ADBC libraries. This release includes [**31
resolved issues**][1] from [**13 distinct contributors**][2].
```

Comment (lidavidm) [reply _posts/2024-11-13-adbc-15-release.md]: I'll fix up the template

File: _posts/2024-11-13-adbc-15-release.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+This is a release of the **libraries**, which are at version
+15.  The **API specification** is versioned separately and is
+at version 1.1.0.
```
Comment (amoeba) [inline _posts/2024-11-13-adbc-15-release.md:33]:
```suggestion
This is a release of the **libraries**, which are at version
15.  The [**API specification**](specification) is versioned 
separately and is at version 1.1.0.
```

Corresponding link added at bottom of doc as a suggestion.

File: _posts/2024-11-13-adbc-15-release.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+can be filed on [GitHub][4], and questions can be directed to GitHub
+or the [Arrow mailing lists][5].
+
+[1]: https://github.com/apache/arrow-adbc/milestone/19
+[2]: #contributors
+[3]: https://github.com/apache/arrow-adbc/blob/apache-arrow-adbc-15/CHANGELOG.md
+[4]: https://github.com/apache/arrow-adbc/issues
+[5]: {% link community.md %}
+
+[opaque]: https://arrow.apache.org/docs/format/CanonicalExtensions.html#opaque
```
Comment (amoeba) [inline _posts/2024-11-13-adbc-15-release.md:94]:
```suggestion
[opaque]: https://arrow.apache.org/docs/format/CanonicalExtensions.html#opaque
[specification]: https://arrow.apache.org/adbc/current/format/specification.html
```

I don't use this linking method but I think I did it right.

File: _posts/2024-11-13-adbc-15-release.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the version 15 release 
+of the Apache Arrow ADBC libraries. This release includes [**31
+resolved issues**][1] from [**13 distinct contributors**][2].
+
+This is a release of the **libraries**, which are at version
+15.  The [**API specification**](specification) is versioned
```
Comment (lidavidm) [inline _posts/2024-11-13-adbc-15-release.md:32]:
```suggestion
15.  The [**API specification**][specification] is versioned 
```

File: _posts/2024-11-13-adbc-15-release.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+or the [Arrow mailing lists][5].
+
+[1]: https://github.com/apache/arrow-adbc/milestone/19
+[2]: #contributors
+[3]: https://github.com/apache/arrow-adbc/blob/apache-arrow-adbc-15/CHANGELOG.md
+[4]: https://github.com/apache/arrow-adbc/issues
+[5]: {% link community.md %}
+
+[opaque]: https://arrow.apache.org/docs/format/CanonicalExtensions.html#opaque
+[specification]: https://arrow.apache.org/adbc/current/format/specification.html
```
Comment (lidavidm) [inline _posts/2024-11-13-adbc-15-release.md:95]:
```suggestion
[specification]: https://arrow.apache.org/adbc/15/format/specification.html
```

File: _posts/2024-11-13-adbc-15-release.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+- R: 0.15.0
+- Rust: 0.15.0
+
+The release notes below are not exhaustive and only expose selected
+highlights of the release. Many other bugfixes and improvements have
+been made: we refer you to the [complete changelog][3].
+
+## Release Highlights
+
+- The BigQuery driver is now properly released to PyPI.
```
Comment (amoeba) [inline _posts/2024-11-13-adbc-15-release.md:49]:
```suggestion
- The BigQuery driver is now properly released to PyPI. See 
  [adbc-driver-bigquery](https://pypi.org/project/adbc-driver-bigquery/).```

**Pull request merged**